src/ensembl/src/content/app/entity-viewer/shared/rest/rest-adaptors/rest-transcript-adaptor.ts

```diff
@@ -0,0 +1,110 @@
+import sortBy from 'lodash/sortBy';
+
+import {
+  TranscriptInResponse,
+  ExonInResponse,
+  FeatureInResponse
+} from 'src/content/app/entity-viewer/shared/rest/rest-data-fetchers/transcriptData';
+
+import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
+import { Strand } from 'src/content/app/entity-viewer/types/strand';
+import { Exon } from 'src/content/app/entity-viewer/types/exon';
+import { CDS } from 'src/content/app/entity-viewer/types/cds';
+
+// transform ensembl rest /overlap data into a transcript data structure
+export const restTranscriptAdaptor = (
+  transcriptId: string,
+  data: FeatureInResponse[]
+): Transcript => {
```

Comment:
Can we omit mentioning the return type here as `buildTranscript` will anyway return a `Transcript`?

Reply:
✔️